### ISSUE TITLE
Nix/NixOS Flake Support

### ISSUE THREAD

**Issue by Cyanide-1** [buttercup/buttercup-desktop#1313]
Hello! I have a suggestion. Could support be added for Nix Flakes? It would allow buttercup to be used on a very good cross-distro linux package format. Thanks!

**Comment by perry-mitchell** [comment]
Absolutely! I've been keeping an eye on Nix but haven't yet jumped in to play around with it yet. I'm familiar with the basic concepts but not more than that. It'd be great to see Buttercup get proper support for Nix.

Would you be interested in helping configure this? Or know where best to start reading into doing this?

**Comment by Cyanide-1** [comment]
> Absolutely! I've been keeping an eye on Nix but haven't yet jumped in to play around with it yet. I'm familiar with the basic concepts but not more than that. It'd be great to see Buttercup get proper support for Nix.
> 
> Would you be interested in helping configure this? Or know where best to start reading into doing this?

im not quite sure what would be the best way to make a flake would be, im not a developer myself or a programmer, just a tech nerd/linux nerd lol

**Comment by perry-mitchell** [comment]
No worries! Well it's good you made an issue as this would have certainly appeared on our radar at some point.

Let's see how it progresses :)
